data/org.freedesktop.portal.Account.xml

```diff
@@ -38,51 +38,28 @@
         Gets information about the user.
 
         Supported keys in the @options vardict include:
-        <variablelist>
-          <varlistentry>
-            <term>handle_token s</term>
-            <listitem><para>
-              A string that will be used as the last element of the @handle. Must be a valid
-              object path element. See the #org.freedesktop.portal.Request documentation for
-              more information about the @handle.
-            </para></listitem>
-          </varlistentry>
-          <varlistentry>
-            <term>reason s</term>
-            <listitem><para>
-              A string that can be shown in the dialog to expain why the information is needed.
-              This should be a complete sentence that explains what the application will do with
-              the returned information, for example: Allows your personal information to be included
-              with recipes you share with your friends.
-            </para></listitem>
-          </varlistentry>
-        </variablelist>
+
+        ``handle_token`` (``s``)
```

Comment:
ftr I found this a lot more readable if these are lists:
```suggestion
        * ``handle_token`` (``s``)
```
Otherwise and because there's no separation between the text and the arguments (which are at the bottom) it's harder to parse than necessary. And that's with me already knowing how it works..

Reply:
This is going to be rendered as a definition list: the indentation is intentional. If you put a list bullet, it breaks the rendering, and it becomes a whole jumble.

Reply:
Rendering without the list:

<details>

![Screenshot from 2023-10-18 15-23-38](https://github.com/flatpak/xdg-desktop-portal/assets/1487352/82eb8b31-9b16-42c4-b998-89f0a210afd8)
</details>

And with the bullet list:
<details>

![Screenshot from 2023-10-18 15-23-14](https://github.com/flatpak/xdg-desktop-portal/assets/1487352/992a5461-f79a-4a22-8248-86756acbd490)
</details>


IMO the latter is better delinated, I don't see any other rendering differences.

Reply:
Weird, but if it works I agree adding the bullet improves things.

Reply:
Just for reference, when using Sphinx and the readthedocs theme as provided by Fedora 39, this is what I get:

<details>
  <summary>No bullet</summary>

![Screenshot from 2023-10-26 22-29-22](https://github.com/flatpak/xdg-desktop-portal/assets/1665/118d6c96-dc3f-448d-b3f0-5e7361048bec)
</details>

<details>
  <summary>With bullet</summary>
  
![Screenshot from 2023-10-26 22-28-31](https://github.com/flatpak/xdg-desktop-portal/assets/1665/a7363da1-2c12-4ff1-868f-9457faff6324)
</details>